styles/focus/catppuccin.user.css

```diff
@@ -0,0 +1,509 @@
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+    .colored-tables() {
+      table {
+        a {
+          color: @blue;
+        }
+        thead th {
+          background: fade(@accent-color, 30%) !important;
+          color: @text;
+        }
+        tr {
+          &:nth-child(odd) {
+            background: @surface0 !important;
+            a {
+              color: @blue;
+            }
+          }
+          &:nth-child(even) {
+            background: fade(@accent-color, 30%) !important;
+            a {
+              color: @blue;
+            }
+          }
+        }
+      }
+    }
+    .violet {
+      background: @mauve !important;
+    }
+    .purple {
+      background: @lavender !important;
+    }
+    .teal {
+      background: @teal !important;
+    }
+    .olive {
+      background: @yellow !important;
+    }
+    .green {
+      background: @green !important;
+    }
+    .pink {
+      background: @pink !important;
+    }
+    .red {
+      background: @red !important;
+    }
+    .blue {
+      background: @blue !important;
+    }
+    .primary {
+      background: @accent-color !important;
+    }
+    .grey {
+      background: @surface0 !important;
+    }
+
+    .calculated-grade {
+      color: @blue !important;
+    }
+    .button {
+      background: @accent-color;
+      color: @base;
+    }
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+    a {
+      color: @blue;
+    }
+    .swift-box-text-value,
+    .swift-box-button {
+      color: @text !important;
+    }
+    .swift-box-options {
+      border-color: @accent-color;
+    }
+    .swift-box-option-container {
+      color: @text;
+      border-color: @accent-color;
+      background: @overlay0 !important;
+      .swift-box-option-highlight {
+        background: @overlay1;
+        color: @text;
+      }
+      .swift-box-option-filter-container {
+        input {
+          background: @mantle !important;
+          border-color: @crust;
+        }
+        .swift-box-option-filter-icon {
+          color: @text !important;
+        }
+        background: @overlay0;
+        border-color: @accent-color;
+      }
+      .swift-box-option-empty-text {
+        color: @text;
+      }
+    }
+    body,
+    .portal::before,
+    .portal,
+    .content-container,
+    .site-middle,
+    .site-container,
+    .sis-package,
+    .document-ready,
+    .site-content-container,
+    .form {
+      background: @base !important;
+      color: @text;
+    }
+    /* fun hack to get the homepage background to style properly */
+    .portal::before {
+      outline: 30rem solid; /* stylelint-disable-line */
+      outline-color: @base;
+      outline-offset: -20rem !important;
+    }
```

Comment:
Elaborate please?

Reply:
no matter what i try there were white bars on both sides of the page so i had to resort to doing this

Reply:
How do they style it?

Reply:
they....don't

the websites styling is confusing and i tried selecting every class, and the bars wouldnt disappear

Reply:
Does a top level `background-color` not fix it?

Reply:
nope